text/0000-mixed-shadow-mode.md

```diff
@@ -167,30 +164,29 @@ added to the component bundle.
 ## Drawbacks
 
 A potential (non-verified) drawback is that existing components may experience a slight performance
-degradation due to the fact that mixed mode requires logic-forking whenever a Shadow DOM API is
-invoked.
+degradation due to the fact that mixed mode may require some logic-forking.
 
 ## Alternatives
 
 Other alternatives have not been considered but contributions to this proposal are welcome.
 
 ## Adoption strategy
 
-Component authors must opt-in when implementing components that prefer native shadow in a mixed
-shadow DOM environment due to observable differences between synthetic mode and native mode. As
-such, it is the responsibility of the component auther to identify and resolve any breaking changes
-for existing components.
-
-Note that the current proposal for mixed mode considers polyfills as all-or-nothing. LWC is not in
-the business of providing workarounds for browser implementation issues, unless it affects the
-functionality of the framework itself.
+When setting `shadowMode` to `any`, it will be the responsibility of the component author to ensure
+that all components in the subtree are compatible with native Shadow DOM. Due to the nature of this
+feature, it is expected that initial usage will start at the leaf components and work its way up the
+component tree. If a component in the subtree does not support native Shadow DOM, the component
+author should work with the other component author to make the appropriate updates.

```

Comment:
So right now we either have

```js
shadowMode = undefined
shadowMode = 'any'
```

Is this right?

Reply:
Yes, that's correct. I didn't make it explicit because it was obvious to me, but I'll add it in there since you asked.